meshmode/discretization/visualization.py

```diff
@@ -441,18 +441,33 @@ def write_high_order_vtk_file(self, file_name, names_and_fields,
                 overwrite=overwrite)
 
     def write_vtk_file(self, file_name, names_and_fields,
-            compressor=None, real_only=False, overwrite=False):
+                compressor=None, real_only=False,
+                overwrite=False, par_namelist=None):
+        r"""
+        Parameters
+        ----------
+        par_namelist
+
+            If par_namelist is given by the caller (and non-empty),
+            it is expected to be a list of filenames wherein the
+            PVTU-format filename and the part source filenames are given.
+            When used by a parallel application, only one rank should
+            write the PVTU file.
```

Comment:
```suggestion
        :arg par_namelist: If par_namelist is given by the caller (and non-empty),
            it is expected to be a list of filenames wherein the
            PVTU-format filename and the part source filenames are given.
            When used by a parallel application, only one rank should
            write the PVTU file.
```
(My packages don't use the numpy docstring convention, at least not yet.)